osdk/src/commands/build/mod.rs

```diff
@@ -207,9 +207,8 @@ fn build_kernel_elf(
         &rustc_linker_script_arg,
         "-C relocation-model=static",
         "-C relro-level=off",
-        // We do not really allow unwinding except for kernel testing. However, we need to specify
-        // this to show backtraces when panicking.
-        "-C panic=unwind",
```

Comment:
Will this prevent the kernel from resuming from test failure(when panicking the kernel)?